multistream.js

```diff
@@ -16,8 +16,8 @@ function multistream (streamsArray, opts) {
   var counter = 0
 
   streamsArray = streamsArray || []
-
   opts = opts || { dedupe: false }
+  levels = opts.levels || levels
```

Comment:
I don't think we should be overwriting the default set of levels. A new closure scoped `levels` should be created instead.

Reply:
@jsumners I agree, good catch!